pkg/controllers/management/rbac/rbac.go

```diff
@@ -160,20 +161,20 @@ func createRole(resourceType, resourceName, namespace, roleAccess, apiVersion st
 	return role, nil
 }
 
-func createRoleBindingForMembers(resourceType, resourceName, namespace, roleAccess, apiVersion string, UID types.UID,
+func createRoleBindingForMembers(resourceType, resourceKind, resourceName, namespace, roleAccess, apiVersion string, UID types.UID,
 	subjects []k8srbacv1.Subject, mgmt *config.ManagementContext) error {
 	roleName, _ := GetRoleNameAndVerbs(roleAccess, resourceName, resourceType)
 	// we can define the rolebinding first, since if it's not already present we can call create. And if it's present then we'll
 	// still need to compare the current members' list
 	sort.Slice(subjects, func(i, j int) bool { return subjects[i].Name < subjects[j].Name })
-	return createRoleBinding(resourceType, resourceName, namespace, roleName, apiVersion, mgmt, UID, subjects)
+	return createRoleBinding(resourceType, resourceKind, resourceName, namespace, roleName, apiVersion, mgmt, UID, subjects)
 }
 
-func createRoleBinding(resourceType, resourceName, namespace, roleName, apiVersion string, mgmt *config.ManagementContext,
+func createRoleBinding(resourceType, resourceKind, resourceName, namespace, roleName, apiVersion string, mgmt *config.ManagementContext,
```

Comment:
`resourceType` is no longer used in this func, or `createRoleBindingForMembers` so it should be dropped (renamed really)

Reply:
Still used by createRoleBindingForMembers; dropped it from createRoleBinding